src/Deprecations.jl

```diff
@@ -0,0 +1,3 @@
+# Deprecated in 0.10.*
+
+@deprecate automorphisms(x::Graph) automorphism_group_generators(x)
```

Comment:
What about `inner_automorphisms_group`, should it also be added here?

Reply:
There is an alias for this.